p2p/switch.go

```diff
@@ -387,7 +387,8 @@ func (sw *Switch) stopAndRemovePeer(p Peer, reason any) {
 }
 
 // reconnectToPeer tries to reconnect to the addr, first repeatedly
-// with a fixed interval, then with exponential backoff.
+// with a fixed interval (approximately 2 minutes), then with
+// exponential backoff (approximately close to 24 hours).
```

Comment:
The problem here for me is the second four (lines 424/426) starts from 0. Anything with power 0 == 1.